src/app/mainpage/mainpage.component.html

```diff
@@ -1,5 +1,40 @@
-<app-header>
-    <span title>This is ShopApp</span>
-    <span description>The best shopping app ever</span>
-</app-header>
-<app-order-form></app-order-form>
+
+<main role="main">
+      <div class="container marketing circles">
+        <div class="row">
+          <div class="col-lg-4">
+            <img class="rounded-circle" src="assets/Justyna.jpg" alt="Generic placeholder image" layout-fill width="140" height="140">
+            <h2>Justyna Podgajna</h2>
+            <p>Donec sed odio dui. Etiam porta sem malesuada magna mollis euismod. Nullam id dolor id nibh ultricies vehicula ut id elit. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Praesent commodo cursus magna.</p>
+            <p><a class="btn btn-secondary" href="#" role="button">Pokaż CV»</a></p>
+          </div>
+          <div class="col-lg-4">
+            <img class="rounded-circle" src="assets/Kacper.jpg" alt="Generic placeholder image" layout-fill width="140" height="140">
```

Comment:
can't we move personal data into objects within the array in ts component and use ngFor directive to display data?

- [ ] create attribute within class, add interface, use ngFor to generatr view and move single object into separate component.